dev-packages/node-integration-tests/suites/tracing-experimental/prisma-orm/test.ts

```diff
@@ -0,0 +1,112 @@
+import { conditionalTest } from '../../../utils';
+import { createRunner } from '../../../utils/runner';
+
+conditionalTest({ min: 16 })('Prisma ORM Tests', () => {
+  test('CJS - should instrument PostgreSQL queries from Prisma ORM', done => {
+    const EXPECTED_TRANSACTION = {
+      transaction: 'Test Transaction',
+      spans: expect.arrayContaining([
+        expect.objectContaining({
+          data: expect.objectContaining({
+            method: 'create',
+            model: 'User',
+            name: 'User.create',
+            'otel.kind': 'INTERNAL',
+            'sentry.origin': 'manual',
```

Comment:
hmm not a hard blocker to merge the tests but @mydea is it expected that the spans all have `'sentry.origin': 'manual'`? Maybe this is an otel<>sentry limitation I'm not aware of.

Reply:
It's because `sentry.origin` is the default, I think this is fine to leave for now and we re-evaluate closer to v8 release.